optaplanner-examples/src/main/java/org/optaplanner/examples/vehiclerouting/optional/benchmark/VehicleRoutingBenchmarkApp.java

```diff
@@ -12,14 +12,8 @@ public VehicleRoutingBenchmarkApp() {
         super(
                 new ArgOption("default",
                         "org/optaplanner/examples/vehiclerouting/optional/benchmark/vehicleRoutingBenchmarkConfig.xml"),
-                new ArgOption("stepLimit",
-                        "org/optaplanner/examples/vehiclerouting/optional/benchmark/vehicleRoutingStepLimitBenchmarkConfig.xml"),
                 new ArgOption("scoreDirector",
                         "org/optaplanner/examples/vehiclerouting/optional/benchmark/vehicleRoutingScoreDirectorBenchmarkConfig.xml")
-        // FIXME Nearby selection for a list variable not yet supported: https://issues.redhat.com/browse/PLANNER-2814
-        // new ArgOption("template",
-        //         "org/optaplanner/examples/vehiclerouting/optional/benchmark/vehicleRoutingBenchmarkConfigTemplate.xml.ftl",
-        //         true)
```

Comment:
Instead of converting these to the new stuff, I decided to deleted them instead.
We have 2 other benchmark configs there, ought to be enough.